edge-detect/coral/docker_setup.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
```

Comment:
Is this file necessary?

Reply:
No, good point. Addressed in https://github.com/IQTLabs/edgetech-coral-detect/pull/1